runtime/doc/pattern.txt

```diff
@@ -477,7 +474,7 @@ More explanation and examples below, follow the links.
 |/\%c|	\%23c	\%23c	in column 23 |/zero-width|
 |/\%v|	\%23v	\%23v	in virtual column 23 |/zero-width|
 
-Character classes {not in Vi}:				*/character-classes*
+Character classes:				*/character-classes*
```

Comment:
:+1: I remember looking at that one and passed it off as okay, but you're right.